### ISSUE TITLE
Use aXe to check for accessibility violations

### ISSUE THREAD

**Issue by cehsu** [unitedvote/united.vote#16]
aXe indicates several accessibility violations, including:
- Buttons must have discernible text
- Elements must have sufficient color contrast
- `<html>` element must have a lang attribute
- Form elements must have labels
- Page must contain one main landmark
- Page must contain a level-one heading
- Content should be contained in a landmark region

**Comment by dsernst** [comment]
Thanks

**Comment by dsernst** [comment]
Is there a link to this test?

**Comment by cehsu** [comment]
aXe offers [browser extensions for Chrome and Firefox](https://axe-core.org/), and has recently come out with a new tool called [aXe-Coconut](https://chrome.google.com/webstore/detail/axe-coconut/iobddmbdndbbbfjopjdgadphaoihpojp?hl=en).

**Comment by dsernst** [comment]
Unclear which / where these are still issues. Can you point to more specifics? 

Maybe helpful to break these into individual issues? Instead of all 7 at once?
